core/modules/layout/layout.module

```diff
@@ -242,14 +223,6 @@ function layout_menu() {
     'type' => MENU_CALLBACK,
   ) + $base;
 
-  $items['admin/structure/layouts/menu/%layout_tempstore_menu_item/condition/edit/%'] = array(
-    'title' => 'Configure condition',
-    'title callback' => 'layout_condition_edit_title',
-    'title arguments' => array(NULL, NULL, 4, 7),
-    'page callback' => 'backdrop_get_form',
-    'page arguments' => array('layout_condition_edit_form', NULL, NULL, 4, 7),
-    'type' => MENU_CALLBACK,
-  ) + $base;
```

Comment:
As noted in https://github.com/backdrop/backdrop-issues/issues/1020#issuecomment-186708336, removal of these callbacks breaks the links to Configure an existing condition after it's been added. We may need to keep these callbacks and the matching forms.